app/src/components/misc/SupportPanel/useSlackInviteVisibility.ts

```diff
@@ -0,0 +1,23 @@
+import { getFunctions, httpsCallable } from "firebase/functions";
+import { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import { getUserAuthDetails } from "store/selectors";
+
+export default function useSlackInviteVisibility() {
+  const [visible, setVisible] = useState(false);
+  const user = useSelector(getUserAuthDetails);
+
+  useEffect(() => {
+    if (!user.loggedIn) return;
+    const showSlackButton = httpsCallable<null, boolean>(getFunctions(), "slackConnect-showSlackButton");
```

Comment:
https://github.com/requestly/requestly/pull/1968#discussion_r1683979551
is it really required to mention the generic types here?